### PR TITLE
Updated jobs to newer versions of Fedora

### PR DESCRIPTION
**Pull request by dralley**
I fully accept that we might want to be more conservative about upgrading the jobs, I just wanted to highlight all of them at first.

Let me know if I should remove any of these changes - happy to do so.

### CODE REVIEW

File: ci/jjb/jobs/projects.yaml [hunk 1]
```diff
@@ -114,23 +114,19 @@
 - project:
     name: pulp-upgrade
     os:
-        - f24
-        - f25
+        - f27
+        - f28
         - rhel6
```
Comment (dralley) [inline ci/jjb/jobs/projects.yaml]: Why RHEL6 ?

File: ci/jjb/jobs/projects.yaml [hunk 2]
```diff
@@ -114,23 +114,19 @@
...
         - rhel7
     pulp_version:
-        - 2.10-stable
         - 2.11-stable
         - 2.12-stable
         - 2.13-stable
         - 2.14-stable
         - 2.15-stable
     upgrade_pulp_version:
-        - 2.15-beta
```
Comment (dralley) [inline ci/jjb/jobs/projects.yaml]: Should I leave this here?

File: ci/jjb/jobs/redmine.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 - job:
     name: 'redmine-bugzilla-automation'
     defaults: ci-workflow-runtest
-    node: 'f25-np'
```
Comment (dralley) [inline ci/jjb/jobs/redmine.yaml]: I'm not sure if this can be moved to RHEL, but if it can, it probably should

**Comment by Ichimonji10** [comment]
Is there any specific reason to drop Fedora 26 at this time? It should be supported until August of this year.

**Comment by dralley** [comment]
@Ichimonji10 According to the (N + 2) + 1 month schedule, it should go EOL next month?

**Comment by Ichimonji10** [comment]
Ah, sorry. Yes, I was assuming the typical 13 month support time. I'm likely wrong.

This change looks fine to me. I'm happy to merge, push to Jenkins, and see what happens. If we're wrong, we can easily revert. Perhaps we should wait until @pcreech finishes his Crane release, though.

**Comment by pcreech** [comment]
Crane beta is out.  Ga is Thursday.

I'm assuming fedora 28 has been added to nodepool?

We still don't have build infra (Koji) for f28 yet due to higher priorities, but you should be able to use f27 packages for the time being.  I'll push harder for f28 koji next week

**Comment by Ichimonji10** [comment]
See: https://github.com/pulp/pulp-ci/pull/527#issuecomment-391831180

**Comment by Ichimonji10** [comment]
For some reason, nodepool is having trouble generating Fedora 28 images. That needs to be fixed before this can be merged. For more information, log in to the nodepool host and look at `journalctl -u nodepoold --since=...`.

**Comment by dralley** [comment]
@Ichimonji10 what's the status on F28 for Nodepool?

**Pull request closed without merging**